frontend/app/build.gradle.kts

```diff
@@ -69,6 +70,17 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    flavorDimensions += "environment"
+    productFlavors {
+        create("realBackend") {
+            dimension = "environment"
+        }
+        create("mockedBackend") {
+            dimension = "environment"
+            versionNameSuffix = ".mocked"  // Optional
+        }
+    }
```

Comment:
This should be moved into client eventually, so that we do not even have to touch rust code when we mock.

Reply:
unfourtunately, currently, even after introducing source sets, we still have to compile the Rust Code as we need the data classes from the generated Uniffi kotlin (like Configuration and Process)